docs/getting-started/first-release.md

```diff
@@ -0,0 +1,199 @@
+---
+template: main.html
+---
+
+# Your first blue-green release
+
+This tutorial shows how Iter8 can be used to release a basic Kubernetes application using a blue-green rollout strategy. The user declaratively describes the desired application state at any given moment. An Iter8 `release` chart assists users who describe the application state at any given moment. The chart provides the configuration needed for Iter8 to automatically deploy application versions and configure the routing to implement a blue-green rollout strategy.
+
+In a blue-green rollout, a percentage of requests are directed to a candidate version of the model. This percentage can be changed over time.
+
+![Blue-green rollout](../tutorials/images/blue-green.png)
+
+In this tutorial, we use the Istio service mesh to distribute inference requests between different versions of a model.
```

Comment:
It's possible to use other service meshes right? Can we mention that here?

Reply:
I removed this comment. Other meshes would require modification to the templates right now.